.github/workflows/build.yml

```diff
@@ -6,8 +6,7 @@ jobs:
   pingora:
     strategy:
       matrix:
-        # TODO: add nightly
-        toolchain: [1.78, 1.72]
+        toolchain: [nightly, 1.72, 1.78]
```

Comment:
Unrelated: can 1.78->1.79 now since we fixed the warning.

Reply:
Done